beacon_node/beacon_processor/src/lib.rs

```diff
@@ -891,6 +902,8 @@ impl<E: EthSpec> BeaconProcessor<E> {
             FifoQueue::new(queue_lengths.rpc_verify_data_column_queue);
         // TODO(das): the sampling_request_queue is never read
         let mut sampling_result_queue = FifoQueue::new(queue_lengths.sampling_result_queue);
+        let mut column_reconstruction_queue =
+            FifoQueue::new(queue_lengths.column_reconstruction_queue);
```

Comment:
we never pop from this queue. Need to add it somewhere in this block based on priority
https://github.com/sigp/lighthouse/blob/b2f7a5befd87ca024915b92fda4f607702af26e9/beacon_node/beacon_processor/src/lib.rs#L1058

Reply:
Addressed in https://github.com/sigp/lighthouse/pull/7588/commits/719df16df6a74c54050b160db90ef92f6dbc6edd